examples/sdk-examples/src/examples/attributeFilter/AttributeParentChildFilterButtonWithPlaceholderExample.tsx

```diff
@@ -36,13 +35,10 @@ const composedLocationFilterPlaceholder = newComposedPlaceholder<IPlaceholder<IA
     cityFilterPlaceholder,
 ]);
 
-const AttributeParentChildFilterButton: React.FC = () => {
-    const [error, setError] = useState<any>();
-
+const AttributeParentChildFilterButtonWithPlaceholder: React.FC = () => {
     const onError: OnError = (...params) => {
-        setError(params);
         // eslint-disable-next-line no-console
-        console.info("AttributeFilterExample onLoadingChanged", ...params);
+        console.info("AttributeFilterExample error found", ...params);
```

Comment:
Nitpick: error found is a bit strange (we did not search for an error). How about just `"AttributeFilterExample error"`?